pkg/tsdb/cloudwatch/cloudwatch_integration_test.go

```diff
@@ -28,17 +31,21 @@ func Test_CloudWatch_CallResource_Integration_Test(t *testing.T) {
 	origNewMetricsAPI := NewMetricsAPI
 	origNewOAMAPI := NewOAMAPI
 	origNewLogsAPI := NewLogsAPI
+	origNewEC2Client := NewEC2Client
 	NewOAMAPI = func(sess *session.Session) models.OAMAPIProvider { return nil }
 
 	var logApi mocks.LogsAPI
 	NewLogsAPI = func(sess *session.Session) models.CloudWatchLogsAPIProvider {
 		return &logApi
 	}
-
+	NewEC2Client = func(provider client.ConfigProvider) models.EC2APIProvider {
+		return fakeEC2Client{}
```

Comment:
Sorry, missed this earlier, but not a big deal if you leave it this way.

This is the "new" mock client which uses testify/mock. Eventually it would be great to replace all of the "old" one and only have this mock left. You'll have to add the `".On("DescribeRegions"....`

```suggestion
		return &mockEC2Client{}
```

In the cases where there is the "old" mock and a "new" one, we could have marked the "old" as deprecated. Not sure why we didn't think of that before...